backend/src/server.ts

```diff
@@ -32,6 +35,34 @@ app.use(express.json())
 app.use(express.urlencoded({ extended: true }))
 app.use(cookieParser(EnvVars.CookieProps.Secret))
 
+
+// Create a connection to the database
+// Create connection to SQLite database
+export const database = new Database('./database.db', (err:any) => {
```

Comment:
See if you can remove the type `:any` here and see if typescript automatically gives it to you.